apps/judicial-system/web/src/components/Logo/Logo.tsx

```diff
@@ -11,14 +11,21 @@ interface Props {
 
 const Logo: React.FC<Props> = ({ defaultInstitution = '' }) => {
   const { user } = useContext(UserContext)
+  const institutionName = user?.institution?.name ?? defaultInstitution
+  const institutionNameArr = institutionName.split(' ')
+  const institutionNameFirstHalf = institutionNameArr.slice(
+    0,
+    institutionNameArr.length - 1,
+  )
 
   return (
     <div className={styles.logoContainer}>
       <Box marginRight={[0, 0, 0, 2]} marginBottom={[0, 0, 1, 0]}>
         <LandWightsLogo />
       </Box>
       <p className={styles.logoText}>
-        {user?.institution?.name ?? defaultInstitution}
+        <span>{institutionNameFirstHalf.toString().replace(',', ' ')}</span>
```

Comment:
Can we not use `word-wrap: break-word` in CSS instead of doing this in JS?

Reply:
The problem with that is that sometimes the institution name is three words and sometimes two, f.x. Lögreglustjórinn á höfuðborgarsvæðinu and Héraðsdómur Reykjaness. We obviously don't want 

Lögreglustjórinn
á
höfuðborgarsvæðinu